lib/unhangout-routes.js

```diff
@@ -467,6 +468,56 @@ module.exports = {
             }
         });
 
+        app.post("/myevents/admin-login-invite/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+            var valid = (
+                "adminInviterName" in req.body &&
+                "eventTitle" in req.body &&
+                "adminInviterEmail" in req.body &&
+                "adminInviteeEmail" in req.body && 
+                "eventID" in req.body
+            )
+
+            if(valid) {
+                //Render the body of the email to send to invitee
+                res.render("admin-login-invite.ejs", {
+                   eventTitle: req.body.eventTitle,
+                   adminInviterName: req.body.adminInviterName,
+                   adminInviteeEmail: req.body.adminInviteeEmail,
+                   adminInviterEmail: req.body.adminInviterEmail,
+                   eventID: req.body.eventID
+                }, function(err, html) {
+                    if (err) {
+                        logger.error("There is an error rendering email body", err);
+                        return res.send(500, "Something went wrong while sending your message. \
+                             Please try again or email us directly at unhangout@media.mit.edu");
+                    }   
+                    var mandrill_client = new mandrill.Mandrill(options.MANDRILL_API_KEY);
+                    var message = {
+                        "html": html,
+                        "subject": "unhangout administrator account",
+                        "from_email": "node@unhangout.media.mit.edu",
+                        "from_name": "Unhangout Team",
+                        "to": [{
+                          "email": req.body.adminInviteeEmail
+                        }],
+                        "headers": {"Reply-To": req.body.adminInviterEmail},
```

Comment:
It's not a *huge* deal, but I'd be more comfortable if we use the email from the Google account (which is stored in the authenticated user's object) rather than a value provided by POST request for this.  A nefarious admin forging the POST request could put any old email as the "reply-to" rather than their own.  This would be a much bigger problem if it was setting the "from" header; but I'd still prefer to constrain the reply-to to one we already know is associated with a google account.